example-deployment/docker/reporting.dockerfile

```diff
@@ -0,0 +1,27 @@
+# FIXME: we are importing from builder because we are installing packages related to ros on bootstrap.
+FROM rmf-web/builder
```

Comment:
Maybe I missed something, but I don't see any rmf messages used so we shouldn't need to base this off of the base rmf image.

Reply:
I thought that `react-components` depends on `rmf-models`, which depends of having the models defined (generate-schemas.py). And since `reporting` depends on `react-components` I thought that it was ok using the rmf-web/builder as the base. But I'll try to split it.

Reply:
Yeah, but `rmf-models` only needs rmf at the generate phase, right now we are committing the generated files to git to make the build step of other packages simpler, at the expense of making it harder to work on both rmf and rmf-web at the same time (because you need to get your rmf changes merged before you can generate new models for rmf-web). Let me know if you think generating all the models on the fly based on the currently sourced rmf is better.

Reply:
To use the same version of docker that the other containers use I'll base on `ros:foxy-ros-base-focal`
https://github.com/osrf/docker_images/blob/master/ros/foxy/ubuntu/focal/desktop/Dockerfile. Is that ok?